root/layout/components/ExternalLinks.js

```diff
@@ -52,6 +54,13 @@ const ExternalLink = ({
     </a>
   );
 
+  if (nonEmpty(entityCredit)) {
+    element = exp.l(
+      '{url} (as {credited_name})',
```

Comment:
Side note: As a translator I would probably not be able to understand what it means without digging the code. But adding a comment like “For linked URL crediting an entity with a different name.” would require MBS-10864. Can be added later on without breaking existing localizations.